### ISSUE TITLE
I want to learn the package agile_grasp!

### ISSUE THREAD

**Issue by hitcww** [atenpas/agile_grasp#7]
I want to learn the package agile_grasp,can you give me SVM training data?
thank you very much!
Best Regards!

-----------------------------------------

hitcww

Harbin Institute of Technology | Master

E：hitcww@163.com

**Comment by atenpas** [comment]
You do not need training data to use the package. What exactly do you want to do?

**Comment by hitcww** [comment]
This package is specially designed to grasp for the Baxter robot, but if I want to use UR5 robot, can the same function be achieved? Do you have a special package for UR5, and can you share it?

**Comment by atenpas** [comment]
No, it's not specially designed for the Baxter robot. UR5 is definitely possible. You should be able to use it on any robot as long as the hand can perform a 2-finger grasp that is similar to how a parallel jaw gripper closes and opens. 

I suggest you also have a look at our newer package: https://github.com/atenpas/gpd (there's a video of the UR5).

**Comment by hitcww** [comment]
ok，thanks
